cli/src/commands/extensions/permissions.rs

```diff
@@ -0,0 +1,159 @@
+use std::path::{Component, PathBuf};
+
+use anyhow::{anyhow, Result};
+use deno_runtime::permissions::PermissionsOptions;
+use serde::{Deserialize, Serialize};
+
+#[derive(Clone, Default, Deserialize, Debug, Serialize)]
+pub struct Permissions {
+    read: Option<Vec<String>>,
+    write: Option<Vec<String>>,
+    run: Option<Vec<String>>,
+    net: Option<Vec<String>>,
+}
+
+// XXX In Deno, `Some(vec![])` actually means "allow all". We do not ever
```

Comment:
I'd recommend changing `do not ever` to `never`, wouldn't want to confuse people in such an important comment.